funnel/assets/js/app.js

```diff
@@ -83,3 +83,16 @@ $(() => {
     $.cookie('timezone', jstz.determine().name(), { path: '/' });
   }
 });
+
+if (
+  navigator.userAgent.match(/(iPhone|Android)/) &&
```

Comment:
This may need `iPad` also.

Reply:
iPad browser doesn't have a scrollbar so we can keep the pace scrollbar